content/source/docs/enterprise-beta/registry/using.html.md

```diff
@@ -0,0 +1,88 @@
+
+![TFE screenshot: the list of available modules](./images/using-modules-list.png)
+
+You can browse the complete list, or shrink the list by searching or filtering.
+
+- The "Providers" drop-down filters the list to show only modules for the selected provider.
+- The search field searches by keyword. This only searches the titles of modules, not READMEs or resource details.
+
+### Viewing Module Details and Versions
+
+Click a module's "Details" button to view its details page. Use the "Versions" dropdown in the upper right to switch between the available versions, and use the Readme/Inputs/Outputs/Dependencies/Resources tabs to view detailed documentation and information about a version.
+
+![TFE screenshot: a module details page](./images/publish-module-details.png)
+
+## Using Modules in Terraform Configurations
+
+In Terraform configurations, you can use any private module from your organization's registry. The syntax for referencing private modules in `source` attributes is `<TFE HOSTNAME>/<TFE ORGANIZATION>/<MODULE NAME>/<PROVIDER>`.
+
+```hcl
+module "vpc" {
+  source = "atlas.hashicorp.com/example_corp/vpc/aws"
+}
+```
+
+If you're using the SaaS version of TFE, the hostname is `atlas.hashicorp.com`; private installs have their own hostnames. The second part of the source string (the namespace) is the name of your TFE organization.
+
+For more details on using modules in Terraform configurations, see ["Using Modules" in the Terraform docs.](/docs/modules/usage.html)
+
+### Usage Examples and the Configuration Designer
+
+Each registry page for a module version includes a usage example, which you can copy and paste to get started.
+
+Alternately, you can use the configuration designer, which lets you select multiple modules and fill in their variables to build a much more useful initial configuration. See [the configuration designer docs](./design.html) for details.
+
+## Running Configurations with Private Modules
+
+### In Terraform Enterprise
+
+TFE can use your private modules during plans and applies with no extra configuration, _as long as the workspace is configured to use Terraform 0.11 or higher._
+
+Note that you can only use private modules from your organization's registry; sharing modules across TFE organizations is not supported.
+
+### On the Command Line
+
+If you're using Terraform 0.11 or higher, you can run configurations that use private modules on the CLI. However, you must provide a valid TFE API token in order to access private modules.
+
+To configure private module access, add a `credentials` block to your [CLI configuration file (`.terraformrc`)](/docs/commands/cli-config.html). The block label for the `credentials` block must be TFE's hostname (either `atlas.hashicorp.com` or the hostname of your private install), and the block body must contain a `token` attribute whose value is a TFE authentication token. You can generate a personal API token from your user settings page in TFE.
```

Comment:
Ah this is what I was looking for in a comment above about the `credentials` block. Maybe a link here would help.